### ISSUE TITLE
Why not using format for parsing? 

### ISSUE THREAD

**Issue by thorek** [webschik/nmbr#2]
Let's say I want to parse 1.234,56 (which with decimal delimiter . and thousands separator , would be 1234.56) I would love to use your library but seems not be supported

**Comment by webschik** [comment]
Hi @thorek !

There is a `parseNumber` function for that https://github.com/webschik/nmbr#parser:

```js
import {parseNumber} from 'nmbr';

parseNumber('1.234,56'); // 1234.56
parseNumber('1,234.56'); // 1234.56

```

I think each function should do just 1 thing: format or parse. So if you need to parse and then format your input number, just use both of them:


```js
import {parseNumber, formatNumber} from 'nmbr';

const input = '1.234,56';
const parsedInput = parseInput(input);
const output = formatNumber(parsedInput, {
    fractionSize: 2,
    thousandDelimiter: ',',
    fractionDelimiter: '.'
}); // 1,234.56

```

**Comment by thorek** [comment]
Yes. As I see the parse function does not take a format string. So given a german format (delimiter . Separator ,) parsing 1,234 would (wrongly) parse 1.234 because it's not aware that the , is a thousands separator, not a decimal delimiter. Right?

**Comment by webschik** [comment]
Just fixed it in v1.3.0, https://github.com/webschik/nmbr#parser
